### DISCUSSION TITLE
Formularios en VUEX

### DISCUSSION THREAD

**Discussion by Texhnolyze47** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#536] [Q&A]
¿Cómo puede enviar información de mi componente formulario a otro componente pedido  y que muestre esta información?

He estado buscando información de como hacer esto pero no logro que la información, del formulario se muestra en mi otro componente.

**Comment by Dfredude** [comment]
Para eso se usa el **_estado global_** o **_global state_** con [**_vuex_**](https://vuex.vuejs.org/#what-is-a-state-management-pattern), lo explico muy bien el Mission Commander en los lives.  Por cierto recuerda que siempre es mejor compartirnos tu codigo  para darnos una mejor idea del problema que tienes.

Si parece que estas usando correctamente el **_global state_** y aun no puedes usar el estado global, tal vez es porque estes usando el elemento de HTML `<form>` y al momento de hacer submit al formulario, recarga la pagina en el navegador! Perdiendo toda la informacion que se haya subido al **_state_**. Entonces tienes dos opciones:

1. Evitar usar `<form>` y manipular el dom para conseguir el `input` **_value_**
2. Agregar un javascript eventlistener para evitar que `<form>` recarge la pagina.

**Comment by MagdielGrande** [comment]
Creo que puedes utilizar en el form @submit.prevent="TuMetodo"  el .prevent hace que no se recargue la página.

**Comment by OmarVenturaP** [comment]
puedes usar el localstorage, allí se guarda en el navegador y lo puedes jalar directamente desde la página de pastelero, aunque el botón sea tipo submit y se recargue la página. Puedes hacerlo instalando el plugin de VUEX persistence.

**Comment by HugoHerreraR** [comment]
usar el localstorage
